os/board/common/README.md

```diff
@@ -0,0 +1,114 @@
+# How to partition a flash into a config
+
+This document provides a way to register a partial block driver in Flash.
+
+## Prepare
+
+To partition Flash, you must first enable ARCH_USE_FLASH. So if new board use FLASH, add 'select ARCH_USE_FLASH' to Kconfig. 
+
+## Contents
+
+> [How to Configure CONFIG](#how-to-configure-config)  
+> [Partition size](#Partition-size)  
+> [Partition type](#partition-type)  
+> [Partition name](#partition-name)  
```

Comment:
When do you use capital character?
Why are `Configure` and `size` different?

Reply:
'configure' is right. I will modify.